### DISCUSSION TITLE
Уведомление об обновлении приложения

### DISCUSSION THREAD

**Discussion by ValeryVerkhoturov** [0niel/university-app#210] [Идеи]
**Проблема**
Нет возможности определить, установлена ли последняя версия ПО на устройстве.

**Решение**
При появлении новой версии приложения в магазине конкретной платформы и новом заходе в приложение единоразово появляется уведомление об обновлении. С предложением можно согласиться и перейти на страницу приложения в магазин приложений (ознакомиться с изменениями и обновиться) или проигнорировать и продолжить пользоваться старой версией.

**Comment by L3odr0id** [comment]
В #169 добавлен функционал, показывающий пользователям новости об обновлениях

**Comment by 0niel** [comment]
Реализовано!
